modules/apps/portal-search/portal-search-web/src/main/java/com/liferay/portal/search/web/internal/suggestions/portlet/SuggestionsPortlet.java

```diff
@@ -91,6 +91,13 @@ public void render(
 		renderRequest.setAttribute(
 			WebKeys.PORTLET_DISPLAY_CONTEXT, suggestionsPortletDisplayContext);
 
+		if (!suggestionsPortletDisplayContext.hasSpellCheckSuggestion() &&
+			!suggestionsPortletDisplayContext.hasRelatedQueriesSuggestions()) {
+
+			renderRequest.setAttribute(
+				WebKeys.PORTLET_CONFIGURATOR_VISIBILITY, Boolean.TRUE);
+		}
```

Comment:
Let's alphabetize:
1) `if` check logic
2) `WebKeys.PORTLET_CONFIGURATOR_VISIBILITY` and `WebKeys.PORTLET_DISPLAY_CONTEXT`